src/binaries/meta-upgrade-09/src/rewrite.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 Datafuse Labs.
+// Copyright 2021 Datafuse Labs
```

Comment:
See https://issues.apache.org/jira/browse/LEGAL-629 for more details. Here we use the interception year of the project.